go.mod

```diff
@@ -147,10 +147,22 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
+require github.com/Azure/azure-storage-blob-go v0.8.0
```

Comment:
Needs a `go mod fmt` in both the root folder, and the `tools` folder.

Reply:
```
$ go mod fmt
go mod fmt: unknown command
Run 'go help mod' for usage.
```

Reply:
`go mod tidy`, sorry.

Reply:
```
lorawan-stack$ go mod tidy
lorawan-stack$ cd tools/
tools$ go mod tidy
tools$ git diff
tools$
```

Reply:
Now I'm curious which tool actually triggers in my local environment the formatting. Anyhow, please do this:

```diff
diff --git a/go.mod b/go.mod
index 3ec5cc5c3..2818fc245 100644
--- a/go.mod
+++ b/go.mod
@@ -44,6 +44,9 @@ replace github.com/go-redis/redis/v8 => github.com/go-redis/redis/v8 v8.4.0
 
 require (
        contrib.go.opencensus.io/exporter/prometheus v0.1.0
+       github.com/Azure/azure-storage-blob-go v0.8.0
+       github.com/Azure/go-autorest/autorest v0.11.22
+       github.com/Azure/go-autorest/autorest/adal v0.9.14
        github.com/PuerkitoBio/purell v1.1.1
        github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
        github.com/TheThingsIndustries/mystique v0.0.0-20200127144137-4aa959111fe7
@@ -147,12 +150,6 @@ require (
        gopkg.in/yaml.v2 v2.4.0
 )
 
-require github.com/Azure/azure-storage-blob-go v0.8.0
-
-require github.com/Azure/go-autorest/autorest/adal v0.9.14
-
-require github.com/Azure/go-autorest/autorest v0.11.22
-
 require (
        cloud.google.com/go v0.81.0 // indirect
        cloud.google.com/go/pubsub v1.3.1 // indirect
```

It's good to go for me aftewards.

Reply:
Done